frontend/javascripts/oxalis/view/left-border-tabs/layer_settings_tab.js

```diff
@@ -785,10 +787,9 @@ class DatasetSettings extends React.PureComponent<DatasetSettingsProps, State> {
 
   render() {
     const { layers } = this.props.datasetConfiguration;
-    const segmentationLayerName = Model.getSegmentationLayerName();
     const layerSettings = Object.entries(layers).map(entry => {
       const [layerName, layer] = entry;
-      const isColorLayer = segmentationLayerName !== layerName;
+      const isColorLayer = getLayerByName(this.props.dataset, layerName).category === "color";
```

Comment:
Minor: There is an `sColorLayer(dataset: APIDataset, layerName: string)` function checking exactly that in the `dataset_accessor.js`.